app/models/report.rb

```diff
@@ -51,6 +51,14 @@ def destroy_reported_item
     mark_as_reviewed
   end
 
+  def mark_as_nsfw
```

Comment:
true but it is always a local user which reports stuff and it happened to me so often.  
Currently the only possibility is to delete the whole post and this is in my opinion to extreme for a missing tag.